diablo/models/sis_section.py

```diff
@@ -800,7 +800,7 @@ def _decorate_course_changes(course):
             ],
         )
         instructor_uids = [i['uid'] for i in course['instructors']]
-        has_obsolete_instructors = set(instructor_uids) != set(course.get('scheduled').get('instructorUids'))
+        has_obsolete_instructors = not set(instructor_uids).issubset(set(course.get('scheduled').get('instructorUids')))
```

Comment:
@paulkerschen - Agree?

Reply:
@johncrossman Sure, as long as losing an instructor isn't a case that we care about capturing.